verification/curator-service/src/util/validate-env.ts

```diff
@@ -0,0 +1,23 @@
+import { CleanEnv, cleanEnv, str, port, url } from 'envalid';
+
+export default function validateEnv(): Readonly<{
+    DATASERVIER_API_URL: string;
```

Comment:
CURATORSERVIER ?

Reply:
Argh, typo re: servier, thanks for the catch.

This is the URL we're using in the HTTP client, so it's for making requests to the data service.